src/components/Editable.js

```diff
@@ -128,6 +128,10 @@ export const Editable = connect()(({ isEditing, thoughtsRanked, contextChain, sh
     }
 
     const oldValueClean = oldValue === EM_TOKEN ? 'em' : ellipsize(oldValue)
+    if (isEM(thoughts) || isRoot(thoughts)) {
+      error(`"${isEM(thoughts) ? 'EM_TOKEN' : 'ROOT_TOKEN'}" cannot be edited.`)
```

Comment:
User-facing text should refer to `EM_TOKEN` as the "**em** context" and `ROOT_TOKEN` as the "home context".